apps/game/src/sandbox/time_warp.rs

```diff
@@ -401,7 +409,7 @@ impl State<App> for TimeWarpScreen {
     }
 }
 
-fn area_under_curve(raw: Vec<(Time, usize)>, width: f64, height: f64) -> Polygon {
+fn area_under_curve(raw: Vec<(Time, usize)>, width: f64, height: f64) -> Result<Polygon, anyhow::Error> {
```

Comment:
https://docs.rs/anyhow/1.0.71/anyhow/#details
I usually `use anyhow::Result` at the top and then return `Result<Polygon>`, which makes the code readability a little better. I've been lazy about error handling through A/B Street and anyhow is perfect for this -- the type of the error doesn't matter anywhere